amethyst_renderer/src/pass/shaded.rs

```diff
@@ -76,7 +82,44 @@ struct PointLight2 {
 
 unsafe impl Pod for PointLight2 {}
```

Comment:
Naming? Doc? I have no idea what this is.

Reply:
Not part of public API. Just implementation detail of the pass.

Reply:
Well. If you're curious. This type us `POD` representation of `PointLight` to pass into shader.